sdk/metricsadvisor/azure-ai-metricsadvisor/src/test/java/com/azure/ai/metricsadvisor/MetricsAdvisorAdministrationClientTestBase.java

```diff
@@ -22,9 +23,29 @@ public abstract class MetricsAdvisorAdministrationClientTestBase extends TestBas
     protected void beforeTest() {
     }
 
+    private HttpClient buildAsyncAssertingClient(HttpClient httpClient) {
+        return new AssertingHttpClientBuilder(httpClient)
+            .skipRequest((ignored1, ignored2) -> false)
```

Comment:
the skipRequest call has this as default value I think now, it doesn't need to be specified.